include/stl2/detail/concepts/object/move_constructible.hpp

```diff
@@ -25,6 +25,10 @@ STL2_OPEN_NAMESPACE {
 	template <class>
 	constexpr bool __addressable = false;
 	template <class T>
+	constexpr bool __addressable<T&> = true;
+	template <class T>
+	constexpr bool __addressable<T&&> = true;
```

Comment:
That definition of `Destructible` changes the result from `true` to `false` for reference types. We don't want that.